openpower/package/openpower-pnor/Config.in

```diff
@@ -112,3 +114,11 @@ config BR2_OPENPOWER_TARGETING_ECC_FILENAME
 config BR2_OPENPOWER_PNOR_XZ_ENABLED
         bool "Enable xz compression in PNOR payloads"
         default n
+
+config BR2_OPENBMC_SYSTEM
+        bool "Build for systems using OpenBMC"
+        default n
+
+config BR2_BUILD_VPNOR
```

Comment:
Yeah, that's probably good.

But I'd *really* like a name for this format, as just re-using other names isn't particularly unique, and overloading terms isn't a good way to clarity.